libcontainer/intelrdt/intelrdt.go

```diff
@@ -203,55 +206,56 @@ type intelRdtData struct {
 	pid    int
 }
 
-// Check if Intel RDT sub-features are enabled in init()
-func init() {
-	// 1. Check if hardware and kernel support Intel RDT sub-features
-	flagsSet, err := parseCpuInfoFile("/proc/cpuinfo")
-	if err != nil {
-		return
-	}
-
-	// 2. Check if Intel RDT "resource control" filesystem is mounted
-	// The user guarantees to mount the filesystem
-	if !isIntelRdtMounted() {
-		return
-	}
-
-	// 3. Double check if Intel RDT sub-features are available in
-	// "resource control" filesystem. Intel RDT sub-features can be
-	// selectively disabled or enabled by kernel command line
-	// (e.g., rdt=!l3cat,mba) in 4.14 and newer kernel
-	if flagsSet.CAT {
-		if _, err := os.Stat(filepath.Join(intelRdtRoot, "info", "L3")); err == nil {
-			catEnabled = true
-		}
-	}
-	if mbaScEnabled {
-		// We confirm MBA Software Controller is enabled in step 2,
-		// MBA should be enabled because MBA Software Controller
-		// depends on MBA
-		mbaEnabled = true
-	} else if flagsSet.MBA {
-		if _, err := os.Stat(filepath.Join(intelRdtRoot, "info", "MB")); err == nil {
-			mbaEnabled = true
+// Check if Intel RDT sub-features are enabled in InitIntelRdt()
+func InitIntelRdt() {
```

Comment:
since the package name is intelrdt, you can just use `Init` as a name (so the call to it will be `interldt.Init()`)

Reply:
> since the package name is intelrdt, you can just use `Init` as a name (so the call to it will be `interldt.Init()`)

@kolyshkin Good suggestion. Thank you!
I have updated the code in commit https://github.com/opencontainers/runc/pull/2678/commits/c2fc85e1d802e3952899d429c10480f0ebfa778d